resources/js/components/Kiosk.vue

```diff
@@ -440,13 +440,23 @@ export default {
                     this.hasError = false;
                     let name = (response.data.name ? response.data.name : "Unknown User");
                     let direction = response.data.punch;
-                    let swalText = (direction === 'in') ? 'Nice to see you, ' + name + '.' : 'Have a great day, ' + name + '!';
+                    let swalText, swalIcon;
+
+                    if (response.data.message != null) {
```

Comment:
No need for the `!= null` in the if condition

Reply:
I swear it didn't work without that but I'll try again.

Reply:
It should at the very least be a strict comparison though (`!==`)